gensim/models/wrappers/wordrank.py

```diff
@@ -95,14 +95,16 @@ def train(cls, wr_path, corpus_file, out_path, size=100, window=15, symmetric=1,
         cmd_del_vocab_freq = ['cut', '-d', " ", '-f', '1', temp_vocab_file]
 
         commands = [cmd_vocab_count, cmd_cooccurence_count, cmd_shuffle_cooccurences]
-        logger.info("Prepare training data using glove code '%s'", commands)
         input_fnames = [corpus_file.split('/')[-1], corpus_file.split('/')[-1], cooccurrence_file]
         output_fnames = [temp_vocab_file, cooccurrence_file, cooccurrence_shuf_file]
 
+        logger.info("Prepare training data using glove code")
```

Comment:
We try to make `info` messages more informative: what data? How much data?

If it's not meant to be seen by end users (internal opaque debug message), it's better to use `debug`.